Create sensor.solax (#9082)
home-assistant/home-assistant.github.io: source/_components/sensor.solax.markdown

---
layout: page
title: "Solax Sensor"
description: "Instructions on how to integrate Solax sensor within Home Assistant."
date: 2019-03-31 04:20
sidebar: true
comments: false
sharing: true
footer: true
logo: solax-logo.png
ha_category: Sensor
ha_release: 0.91
ha_iot_class: Local Polling
---

> The `solax` component connects home-assistant to Solax solar power inverters. Solax inverters may be connected to a home Wi-Fi network and expose a REST api. This component retrieves information such as photo voltaic power production, battery levels and power, and how much power is being fed back into the grid.

Reviewer: ```suggestion
The `solax` component connects home-assistant to Solax solar power inverters. Solax inverters may be connected to a home Wi-Fi network and expose a REST API. This component retrieves information such as photovoltaic power production, battery levels and power, and how much power is being fed back into the grid.
```

## {% linkable_title Configuration %}

To use the Solax sensors in your installation, add the following to your configuration.yaml file:

> {% raw %}

Reviewer: No need for raw here
```suggestion
```

```yaml
# Example configuration.yaml entry
sensor:
  - platform: solax
    ip_address: IP_ADDRESS
```

> {% endraw %}

Reviewer: ```suggestion

```

{% configuration %}

> 

Reviewer: Unneeded empty line
```suggestion
```

ip_address:
  description: The IP address of your Solax system.
  required: true
  type: string
{% endconfiguration %}

### {% linkable_title Full configuration sample %}

> {% raw %}

Reviewer: ```suggestion
```

A full configuration entry would look like the sample below.

```yaml
# Example configuration.yaml entry
sensor:
  - platform: solax
    ip_address: 192.168.0.3
```

> {% endraw %}

Reviewer: ```suggestion
```
